packages/webview_flutter/webview_flutter_wkwebview/lib/src/webkit_webview_controller.dart

```diff
@@ -216,6 +216,34 @@ class WebKitWebViewController extends PlatformWebViewController {
           return decisionCompleter.future;
         }
       },
+      runJavaScriptAlertDialog: (String message) async {
+        final Future<void> Function(String message)? callback =
+            _onJavaScriptAlertDialogCallback;
```

Comment:
This needs to be `weakThis.target?._onJavaScriptAlertDialogCallback`. The methods below also need to be updated.